TROUBLESHOOTING.md

```diff
@@ -98,19 +98,6 @@ that prevents it from responding to new requests coming from the editor. If you
 report [here](https://github.com/Shopify/vscode-ruby-lsp/issues/new?labels=bug&template=bug_template.yml) including the
 steps that led to the server getting stuck.
 
-### Developing on containers
```

Comment:
Let's keep a link to the docs in troubleshooting because we link directly to this document from an error dialog.

Something like

```
### Developing on containers

See the [documentation]().
```

Reply:
:+1: